### DISCUSSION TITLE
Java

### DISCUSSION THREAD

**Discussion by Swapnikavinukonda** [Swapnikavinukonda/java_files#9] [Q&A] [answered]
What are Features of Java?

**Comment by Parthasaradhi19** [comment]
Platform Independence. 
Automatic Memory Management. 
Security. 
Rich API. 
Multithreading. 
High Performance.

**Comment by Swapnikavinukonda** [comment, marked as answer]
it is simple and easy to learn.
it is object oriented programming language.
